src/core/libmaven/mzSample.cpp

```diff
@@ -308,6 +315,20 @@ void mzSample::parseMzML(const char *filename)
 	}
 }
 
+void mzSample::parseMzMLInjectionTimeStamp(xml_node experimentRun)
```

Comment:
use reference instead of just copying the entire object

Reply:
@rish9511 Just trying to be consistent here. There are other parseMLML functions also.

Reply:
that's also bad though. okay, your call

Reply:
@rish9511 Updated the code